### PR TITLE
1186445 - Ensure that the install distributor install_path already exists before we start working with it

### PR DESCRIPTION
**Pull request by barnabycourt**

### CODE REVIEW

**Comment by jlsherrill** [comment]
tested on a patched 2.5.1 pulp and it worked well for me!

**Comment by stbenjam** [comment]
Me too, thanks! :+1:

File: pulp_puppet_plugins/pulp_puppet/plugins/distributors/installdistributor.py [hunk 1]
```diff
@@ -217,6 +218,20 @@ def _rename_directory(unit, destination, names):
         after = os.path.join(destination, unit.unit_key['name'])
         shutil.move(before, after)
 
+    def _ensure_destination_dir(self, destination):
+        """
+        Ensure that the directory specified by destination exists
+
+        :param destination: The directory to create
```
Comment (mhrivnak) [inline pulp_puppet_plugins/pulp_puppet/plugins/distributors/installdistributor.py]: I would just add the words "full path to..." to be explicit.

**Comment by mhrivnak** [comment]
LGTM

**Pull request merged**